.github/configuration/upgrade-test-configuration.yml

```diff
@@ -65,4 +65,24 @@ upgrade-version: [[
     version: target.latest,
     upgrade-type: major 
   }
+],
+[
+  {
+    version: 13.7,
+    upgrade-type: null
+  },
+  { 
+    version: target.latest,
+    upgrade-type: major 
+  }
+],
+[
+  {
+    version: 13.8,
+    upgrade-type: null
+  },
+  { 
+    version: target.latest,
```

Comment:
we should also add minor upgrade for 13.7/13.8 as well

Reply:
included minor version upgrade step in each upgrade path